### PR TITLE
docs: rework bug report template

### PR DESCRIPTION
**Pull request by ephys**
I've found that too often, people don't think a problem is dialect related and therefore don't provide any information regarding which database they're using, which connector, etc

This PR changes our template to always request that information

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (ephys) [reply .github/ISSUE_TEMPLATE/bug_report.md:103]: I'm hoping they can figure it out, otherwise the comment is going to be very long given how many different connectors we support

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (ephys) [reply .github/ISSUE_TEMPLATE/bug_report.md:105]: It used to be a subcategory of 'Bug Report Checklist' but as there is only one question, I promoted it and removed 'Bug Report Checklist'

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (WikiRik) [reply .github/ISSUE_TEMPLATE/bug_report.md:100]: The comment

Comment (ephys) [reply .github/ISSUE_TEMPLATE/bug_report.md:100]: I just added it back to `Environment` as that's a very important section

**Review by WikiRik — APPROVED** [review]

**Comment by WikiRik** [comment]
Let's wait until the new CI PR has been merged and then MariaDB will pass again

**Pull request merged**
